reference/providers/cf.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: single
+title:  "Cloud Foundry"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+If you are not familiar with Cloud Foundry (CF) or any of the terms used below, please consult [Cloud Foundry's reference documentation](https://docs.cloudfoundry.org).
+
+## Resource mapping
+
+### Account
+In CF, an account maps to a user account on a CF foundation. This user account is provided via configuration to Spinnaker, and Spinnaker’s privileges are determined by the account’s permissions; for example, Spinnaker can manage apps across all of the orgs and spaces which its account can access.
+
+A Spinnaker instance can use multiple CF accounts to access one or multiple CF foundations.
+
+### Load Balancer
+A Spinnaker load balancer maps to a CF route, which is associated with an app and does not have a separate resource name. The name of the route is the fully-qualified path, including host, domain, port, and path. A load balancer is created as part of a Server Group definition.
```

Comment:
@fifthposition 
I assume by "app" you're not referring to a Spinnaker application, right?

Reply:
Fixed. Good catch. Thanks!